app/models/transaction_type.rb

```diff
@@ -36,7 +38,7 @@ class TransactionType < ApplicationRecord
 
   scope :active, -> { where(archived_at: nil) }
   scope :debits, -> { active.where(operation: :debit) }
-  scope :credits, -> { active.where(operation: :credit) }
+  scope :credits, -> { active.where(operation: :credit).where.not(name: DISREGARDED_BENEFITS) }
```

Comment:
I think we need to think about how housing benefit transactions map onto our existing data model.

It makes sense that housing benefit is a regular transaction for the purposes of obtaining a financial eligibility result from CFE, but I'm not sure it makes sense across the application.

We already have an `excluded_benefits` transaction type, which seems to correspond to the domain concept of "Disregarded benefits".

It doesn't seem like this was implemented with the intention of there being multiple varieties of `excluded_benefits` transaction types, but I'm not really sure what the original intention was anyway.

It looks like the concept was introduced in #1425, and iterated on in #1447 and #1575 —but it's hard to glean much context from the commit messages.

Given `housing_benefit` is a type of `excluded_benefit`, I'm not sure what else we need to consider for this to go live.

- We want to submit it to CFE as a regular transaction
- We don't want to display it in forms alongside other income types

But, 
- Do we want to display it as a category for bank transactions?
- Are there things we need to consider in terms of means reports etc?

I'm not sure who knows the answers to these questions, so I'm interested to hear if anyone can shed some light on any of this!

Reply:
> Do we want to display it as a category for bank transactions?

I do not believe so. There is "supposed to be" automatic categorisation of bank transactions as a state benefit of type housing_benefit if the description of the bank transaction contains the works "DWP" and "HB", however, I do not believe it is working as intended following some commentary from StephenR a few weeks ago. But it may need wider discussion with the BA and others

Reply:
> Are there things we need to consider in terms of means reports etc?

That will hopefully come out of [AP-3498](https://dsdmoj.atlassian.net/browse/AP-3498) and the [design](https://docs.google.com/document/d/1LbPEoEM3tlqRt3KeCmD7hNsRi9dvzLzgQEMnWIbOsjI/edit)

Reply:
> We want to submit it to CFE as a regular transaction

Yes, that is already handled by Apply's `CFE::SubmissionManager` and is accounted for in CFE (excluded from income, but negated from gross housing costs if necessary)